tembo-operator/src/cloudnativepg/cnpg.rs

```diff
@@ -1047,6 +1047,10 @@ pub async fn reconcile_cnpg(cdb: &CoreDB, ctx: Arc<Context>) -> Result<(), Actio
             None
         }
     };
+    // If we can't find the existing primary pod, returns a requeue
+    let primary_pod_cnpg = cdb
+        .primary_pod_cnpg_ready_or_not(ctx.client.clone())
```

Comment:
Ok I was worried that this would be an issue, but since we are calling `primary_pod_cnpg_ready_org_not` which calls this function https://github.com/tembo-io/tembo/blob/cac0c8960e4312abfe430a7f793987bf9b4dd832/tembo-operator/src/controller.rs#L525 we should be fine.  Since it will return the pod even if it's not ready.

This part is GTG